### PR TITLE
Add test case to validate PMU is disbaled for ARM64

### PR DESCRIPTION
**Pull request by sharsonia**

### CODE REVIEW

File: microsoft/testsuites/core/timesync.py [hunk 1]
```diff
@@ -351,3 +351,24 @@ def timesync_chrony(self, node: Node) -> None:
         chrony.check_sources_and_stats()
         # 5. Check chrony tracking.
         chrony.check_tracking()
+
+    @TestCaseMetadata(
+        description="""
+        This test is to check "initcall_blacklist=arm_pmu_acpi_init"
+        kernel parameter for ARM64 images.
+        """,
```
Comment (LiliDeng) [inline microsoft/testsuites/core/timesync.py:364]: can you put more context here, like why need blacklist this, is it a temp fix?

File: microsoft/testsuites/core/timesync.py [hunk 2]
```diff
@@ -351,3 +351,24 @@ def timesync_chrony(self, node: Node) -> None:
...
+        lscpu = node.tools[Lscpu]
+        arch = lscpu.get_architecture()
+        if arch is not CpuArchitecture.ARM64:
+            raise SkippedException(
+                f"This test case does not support {arch}."
+                "This validation is only for ARM64."
+            )
+        # Check pmu is disabled in cmdline for arm64 images
+        cat = node.tools[Cat]
+        result = cat.run("/proc/cmdline", force_run=True)
```
Comment (LiliDeng) [inline microsoft/testsuites/core/timesync.py:377]: please double check whether all distro put blacklist in this file

Comment (sharsonia) [reply microsoft/testsuites/core/timesync.py:377]: /proc/cmdline will have boot params which is what we are verifying that the boot params have this string - "initcall_blacklist=arm_pmu_acpi_init"

/proc/cmdline exists in Ubuntu, Redhat (and CentOS). Let me verify for Alma and other such

Comment (sharsonia) [reply microsoft/testsuites/core/timesync.py:377]: > please double check whether all distro put blacklist in this file

@LiliDeng

Results from various runs
https://microsoft.visualstudio.com/LSG/_build/results?buildId=63133136&view=results
https://microsoft.visualstudio.com/LSG/_build/results?buildId=63114483&view=results
https://microsoft.visualstudio.com/LSG/_build/results?buildId=63146846&view=results

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**